javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md

```diff
@@ -37,25 +23,26 @@ Once you have created an element you need to add it to the HTML document, or it
 In this example we will use the `getElementById` to select the parent, then our `initializeElement` function to create a heading and then use `appendChild` to add it to the document:
 
 ```javascript
-var container = document
-  .getElementById('container');
+let container = document.getElementById(
+  "container"
+);
 
 function initializeElement(element, text) {
-  var newElement = document
-    .createElement(element)
-  newElement
-   .appendChild(document.
-      createTextNode(text))
-  return newElement
+  var newElement = document.createElement(
```

Comment:
```suggestion
  let newElement = document.createElement(
```
shoud we update this `var` to `let`?

Reply:
Yeah, why not.

The rule can be, always use `let`, unless to demonstrate the nuances of `var` or `const`